packages/cacti-plugin-consortium-static/src/main/typescript/repository/new-consortium-provider.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Logger,
+  LogLevelDesc,
+  LoggerProvider,
+} from "@hyperledger/cactus-common";
+
+import { Checks, IAsyncProvider } from "@hyperledger/cactus-common";
+import { ConsortiumDatabase } from "@hyperledger/cactus-core-api";
+
+import {
+  DefaultApi as ConsortiumNewApi,
+  GetConsortiumJwsResponse,
+} from "../generated/openapi/typescript-axios/api";
+
+export interface INewConsortiumProviderOptions {
+  logLevel?: LogLevelDesc;
+  apiClient: ConsortiumNewApi;
+}
+
+export class NewConsortiumProvider
```

Comment:
@eduv09 Please rename this as well, I'm guessing it's a leftover from before we came up with the current name.